development/Writing-a-plugin.md

```diff
@@ -54,24 +54,24 @@ class MySuperCoolPlugin(manager: PluginManager) extends PluginImpl(manager) {
 
-The `getRequirements` method is used by the framework to check what your plugin needs and to ask the user to specify which services or data he wants to give to your plugin. Here is a example:
+The `getRequirements` method is used by the framework to check what your plugin needs and to ask the user to specify which services or data he wants to give to your plugin. Here is an example:
 
 ```
 private val require = new Requirements
 private val twitchChatInput = require.input.twitchChat("reqTwitch", "A twitch channel", false)
 private val nameToSayHelloTo = require.parameter.string("reqHello", "Your name", false)
 ```
 
-This code requires reading-access to a twitch livestream chat and a parameter ("*a name to say hello to"*). Note: You can obviously not specify, which channel the user might choose - but you can be sure, that you will get a working input. This is ensured by the framework. Please note, that the requirements element already exists, if you extend `PluginImpl`.
+This code requires read-access to a twitch livestream chat and a parameter ("*a name to say hello to"*). Note: You can obviously not specify which channel the user might choose - but you can be sure that you will get a working input. The framework takes care of that. Please note that the requirements element already exists if you extend `PluginImpl`.
 
-After this, the last step ist the `start` method. And it is as easy as you might think: Just add your own logic now, using the required parameters of your Requirement object. Here is a short example: This code will simply print out all messages from a twitch chat in the console. To access the chat, your required variable is used with the `getValue` method.
+After this, the last step is the `start` method. And it is as easy as you might think: Just add your own logic now, using the required parameters of your Requirement object. Here is a short example: This code will simply print out all messages from a twitch chat to the console. To access the chat, your required variable is used with the `getValue` method.
 
 ```
 twitchChatInput.getValue.registerMessageHandler(msg => println(msg))
 ```
 
 Of course, you can also add own classes and infrastrucutre by now - the important steps are done, happy coding!
 
-*One last note: If your testing your plugin the first time, a full reload might be needed. Use the custom task `[Advanced] Full Reload and run ChatOverlfow` to do so. Afters this, you can configure the framework to start your plugin using the [CLI](usage/Using-the-CLI.md)!*
\ No newline at end of file
+*One last note: If you'r testing your plugin the first time, a full reload might be needed. Use the custom task `[Advanced] Full Reload and run ChatOverlfow` to do so. Afters this, you can configure the framework to start your plugin using the [CLI](usage/Using-the-CLI.md)!*
```

Comment:
s/you'r/you are
s/the first time/for the first time
s/ChatOverlfow/ChatOverflow